packages/nuxt/src/app/nuxt.ts

```diff
@@ -190,6 +190,14 @@ export interface ObjectPlugin<Injections extends Record<string, unknown> = Recor
    * @default false
    */
   parallel?: boolean
+  /**
+   * Await for plugin to be finished before running this plugin.
+   */
+  dependsOn?: string[]
```

Comment:
Would be nice to provide strong types for this, as we do for layout names. (Can be a future pr.)

Reply:
done !